genref/output/md/kube-scheduler-config.v1beta2.md

```diff
@@ -322,7 +322,7 @@ The default strategy is LeastAllocated with an equal &quot;cpu&quot; and &quot;m
 <td>
    <p>DefaultConstraints defines topology spread constraints to be applied to
 Pods that don't define any in <code>pod.spec.topologySpreadConstraints</code>.
-<code>.defaultConstraints[&lowast;].labelSelectors</code> must be empty, as they are
+<code>.defaultConstraints[&amp;lowast;].labelSelectors</code> must be empty, as they are
```

Comment:
nit: Is this correct?

Reply:
No. This is unexpected. Sigh...

I was trying to substitute '*' by '&lowast;', however, the goldmark package converted that '&' to '&amp;'.

After another round of tuning, I'm now inclined to give up this substitution.